news_crawler/src/news_crawler.py

```diff
@@ -3,6 +3,7 @@
 import uuid
 import os
 import boto3
+from boto3.dynamodb.conditions import Attr
```

Comment:
you put a lot of stuff into the file. I would split it a little bit. 
Maybe a file for crawling the data and a file for saving the data. 
Than you can call it from this file

Reply:
tracked in issue 77